programs/compressed-pda/src/verify_state.rs

```diff
@@ -152,59 +152,105 @@ pub fn sum_check(
     }
 }
 
+/// If signer seeds are not provided, invoking program is required.
+/// If invoking program is provided signer seeds are required.
```

Comment:
basically always `signer_seeds` and `invoking_program` are required ?

Reply:
when doing a cpi yes